origin-js/src/contractInterface/users/v01_adapter.js

```diff
@@ -122,7 +122,7 @@ export default class V01_UsersAdapter {
    * @return {Promise<string|null>}
    * @private
    */
-  async _getIdentityIpfsHash(address) {
+  async _getIdentityIpfsHash(address = '') {
```

Comment:
As opposed to setting a default value in this method, I would
1. Add a safeguard that returns null if !address. e.g.
if (!address) {
  return null
}

2. Add a similar safeguard in the async get(address) method above at line 84:
if (!address) {
  return false
}

Reply:
@franckc if you wouldn't mind, I'm mid-other-branch right now dealing with mobile. I'll come back to this a bit later if you haven't gotten to it. Thanks!